app/jobs/audit/save_to_persistent_storage.rb

```diff
@@ -0,0 +1,11 @@
+module Audit
+  class SaveToPersistentStorage < ApplicationJob
```

Comment:
Just a little detail, all existing classes are called `SomethingSomethingJob` so at least to keep consistency I'd rename this to `SaveToPersistentStorageJob` and the file to `save_to_persistent_storage_job.rb`